forui/test/src/widgets/tooltip/tooltip_test.dart

```diff
@@ -0,0 +1,287 @@
+              longPressExitDuration: duration,
+              tipBuilder: (context, style, _) => const Text('tip'),
+              child: FButton(
+                onPress: () {},
+                label: const Text('button'),
+              ),
+            ),
+          ),
+        );
+
+        await tester.longPress(find.byType(FButton));
+        await tester.pumpAndSettle();
+
+        expect(find.text('tip'), findsOneWidget);
+
+        await tester.pumpAndSettle(kLongPressTimeout + kPressTimeout + duration);
+
+        expect(find.text('tip'), findsNothing);
+      });
+
+      testWidgets('long-press and re-long-press shows resets hoverEnterDuration', (tester) async {
+        const duration = Duration(milliseconds: 1000);
+
+        await tester.pumpWidget(
+          TestScaffold(
+            data: FThemes.zinc.light,
+            child: FTooltip(
+              longPressExitDuration: duration,
+              tipBuilder: (context, style, _) => const Text('tip'),
+              child: FButton(
+                onPress: () {},
+                label: const Text('button'),
+              ),
+            ),
+          ),
+        );
+
+        await tester.longPress(find.byType(FButton));
+        await tester.pumpAndSettle();
+
+        expect(find.text('tip'), findsOneWidget);
+        await tester.pumpAndSettle(kLongPressTimeout + kPressTimeout + const Duration(milliseconds: 100));
+
+        await tester.longPress(find.byType(FButton));
+        await tester.pumpAndSettle();
+
+        expect(find.text('tip'), findsOneWidget);
+        await tester.pumpAndSettle(kLongPressTimeout + kPressTimeout + const Duration(milliseconds: 1000));
+
+        expect(find.text('tip'), findsNothing);
+      });
+    });
+
+    group('desktop', () {
+      testWidgets('manual does nothing', (tester) async {
+        debugDefaultTargetPlatformOverride = TargetPlatform.linux;
+
+        final controller = FTooltipController(vsync: const TestVSync());
+
+        await tester.pumpWidget(
+          TestScaffold(
+            data: FThemes.zinc.light,
+            child: FTooltip.manual(
+              controller: controller,
+              tipBuilder: (context, style, _) => const Text('tip'),
+              child: FButton(
+                onPress: () {},
+                label: const Text('button'),
+              ),
+            ),
+          ),
+        );
+
+        final gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
+        await gesture.addPointer(location: Offset.zero);
+        addTearDown(gesture.removePointer);
+        await tester.pump();
+
+        await gesture.moveTo(tester.getCenter(find.byType(FButton)));
+        await tester.pumpAndSettle(const Duration(seconds: 1));
+
+        expect(find.text('tip'), findsNothing);
+
+        debugDefaultTargetPlatformOverride = null;
+      });
+
+      testWidgets('hover shows tooltip', (tester) async {
+        debugDefaultTargetPlatformOverride = TargetPlatform.linux;
```

Comment:
Same here, does the platform matter? Though it could be interesting to see if this could be used to avoid the slight variances between how text is rendered between platforms.

Reply:
The platform matters for Resizable since there's a difference in hit area for the different platforms.